### ISSUE TITLE
Image transformation on each save

### ISSUE THREAD

**Issue by ti0rafa** [gumlet/php-image-resize#129]
Hello,

I notice that after saving an image more then once the image kept changing its colors.

```
$Logo = new ImageResize('test.png');
$Logo->resizeToBestFit(260, 260, true);
$Logo->save('testA.png');
$Logo->save('testB.png');
$Logo->save('testC.png');
```

The original image is saved correctly in "testA" but after that it keep changing colors, the original image was yellow but started changing after the first save, "testB" was a bit orange and "testC" was almost red.

**Comment by adityapatadia** [comment]
That might be due to gamma correction. I will make required changes to disable it.

**Comment by adityapatadia** [comment]
Closed by #132
